src/Widgets/Panel.vala

```diff
@@ -247,12 +247,21 @@ public class Wingpanel.Widgets.Panel : Gtk.EventBox {
     private void add_indicator (Indicator indicator) {
         var indicator_entry = new IndicatorEntry (indicator, popover_manager);
 
-        switch (indicator.code_name) {
-            case Indicator.APP_LAUNCHER:
+        // Workaround until both indicators get their own position prop
```

Comment:
```suggestion
        // TODO Workaround until both indicators get their own position prop
```

Please explicit TODO so that this comment won't be buried into other normal comments.